UPGRADE.md

```diff
@@ -1,5 +1,10 @@
 # Upgrade to 2.11
 
+## Deprecated the functionality of dropping client connections when dropping a database
+
+The corresponding `getDisallowDatabaseConnectionsSQL()` and `getCloseActiveDatabaseConnectionsSQL` methods
+of the `PostgreSqlPlatform` class have been deprecated.
```

Comment:
Why only Postgres? #4225 is about Oracle.

Reply:
The deprecation covers all platforms. Specifically, the ones that override `SchemaManager::dropDatabase()` and implement error handling there. Those include Oracle, Postgres, and others.

But only Postgres has public methods that are used in such implementation. Those methods are also deprecated.